src/rules/preferFunctionOverMethodRule.ts

```diff
@@ -66,10 +66,11 @@ class Walker extends Lint.RuleWalker {
                 const { name } = node as ts.MethodDeclaration;
                 const usesThis = this.withThisScope(
                     name.kind === ts.SyntaxKind.Identifier ? name.text : undefined,
-                    () => super.visitNode(node));
-                if (!usesThis &&
-                    node.parent!.kind !== ts.SyntaxKind.ObjectLiteralExpression &&
-                    this.shouldWarnForModifiers(node as ts.MethodDeclaration)) {
+                    () => super.visitNode(node),
+                );
+                if (!usesThis
+                        && node.parent!.kind !== ts.SyntaxKind.ObjectLiteralExpression
```

Comment:
If someone wanted to enforce the

```ts
1 +
    2 +
    33333 +
    4
```

style, it wouldn't make sense to call it `align`, since the operators are all at different x-position.
Also, if there is a single operator,

```ts
1
    + 2
```

it isn't really aligned with anything.
Full `indent` (#581) amount checking would replace `align` anyway, right?